setup.cfg

```diff
@@ -41,8 +41,8 @@ testpaths =
 
 [coverage:run]
```

Comment:
I don't this this omit is in the right section.

Reply:
I'll try `coverage ... --debug=trace` later which should help in finding the problem, as per https://coverage.readthedocs.io/en/latest/cmd.html#diagnostics-coverage-debug